app/Models/Multiplayer/Score.php

```diff
@@ -19,14 +19,40 @@
  */
 namespace App\Models\Multiplayer;
 
-class Score extends Model
+use App\Models\Score\Model as BaseModel;
```

Comment:
is this just to get the score calculation functions? Perhaps better to move the functions to a trait.